loops-arrays-objects.js

```diff
@@ -132,4 +175,22 @@ const students = [
 // ==========================================
 
 
+//==================================================================================================================
+//=========IK HEB GEEN TIJD GEHAD OM DEZE OPDRACHT AF TE MAKEN EN KWAM ER NIET UIT VOOR 23.59, =====================
+//=========ALS JE HET ANTWOORD WEET BEN IK BENIEUWD HOE JE DIT HEB GEDAAN, GROETEN EN BEDANKT. =====================
+//==================================================================================================================
+//       ||      ||      ||      ||      ||      ||      ||      ||      ||
+//       ||      ||      ||      ||      ||      ||      ||      ||      ||
+//       ||      ||      ||      ||      ||      ||      ||      ||      ||
+//     \    /  \    /  \    /  \    /  \    /  \    /  \    /  \    /  \    /
+//      \  /    \  /    \  /    \  /    \  /    \  /    \  /    \  /    \  /
+//       \/      \/      \/      \/      \/      \/      \/      \/      \/
+
+
+// for (let i = 0; i < students.length; i++) {
+//     switch (students[i].zipCode)
+//         case: students[i].neighborhood
+//         students[i].neighborhood = "Pijlsweerd"
+//     console.log(students)
+// }
```

Comment:
for (let i = 0; i < students.length; i++) {
    switch (students[i].zipCode) {
        case '3513':
            students[i].neighborhood = 'Pijlsweerd';
            break;
        case '3514':
            students[i].neighborhood = 'Vogelenbuurt';
            break;
        case '3512':
            students[i].neighborhood = 'Vogelenbuurt';
            break;
        case '3531':
            students[i].neighborhood = 'Vogelenbuurt';
            break;
        case '3572':
            students[i].neighborhood = 'Vogelenbuurt';
            break;
        case '3581':
            students[i].neighborhood = 'Vogelenbuurt';
            break;
        case '3583':
            students[i].neighborhood = 'Vogelenbuurt';
            break;
    }
}

console.log(students);